src/srb2kart/network/info/info.go

```diff
@@ -0,0 +1,238 @@
+package info
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+
+	"GoBun/functional/strings"
+)
+
+var GetServerInfoPacket = [...]byte{
+	0x58, 0x46, 0x23, 0x01,
+	0x00,
+	0x00,
+	0x0C,
+	0x00,
+
+	0x01,
+	0x1f, 0x02, 0x00, 0x00,
+}
+
+type PacketType uint8
+
+const (
+	PacketTypeServerInfo PacketType = 0x0D
+	PacketTypePlayerInfo PacketType = 0x0E
+)
+
+type packetHeader struct {
+	Checksum   uint32
+	Ack        uint8
+	Ackret     uint8
+	PacketType PacketType
+	_          [1]byte
+}
+
+type serverInfoPacketRaw struct {

```

Comment:
I'd still put the server info and player info stuff in separate files, and the common types in another.

Reply:
like, if you were to create the path to the file, you'd end up with `.../network/info/info.go`, which is redundant. There's more useful information about the code that you can put in the filename to organize the code better.